src/bun.js/api/bun/subprocess.zig

```diff
@@ -2157,7 +2161,22 @@ pub const Subprocess = struct {
                                             return .zero;
                                         };
                                     },
-                                    else => {},
+                                    .inherit => {
+                                        stdio_inherits.append(bun.default_allocator,
+                                            @intCast(i)
+                                        ) catch {
+                                            globalThis.throwOutOfMemory();
+                                            return .zero;
+                                        };
+                                    },
+                                    else => {
```

Comment:
Maybe use .ignore here explicitly, so if the enum changes there'll be an automatic compile-time error.